packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -227,6 +234,27 @@ export async function activate(context: vscode.ExtensionContext) {
     sfdxProjectOpened = files && files.length > 0;
   }
 
+  // Set environment variable to add logging for VSCode API calls
+  process.env.SFDX_SET_CLIENT_IDS = CLIENT_ID;
```

Comment:
Process environment will add logging to CLI commands executed through the command palette.